lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -911,9 +983,6 @@ instance Buildable addr => Buildable (ConnectionId addr) where
 instance Buildable LocalAddress where
     build (LocalAddress p) = build p
 
-instance Buildable W.ChimericAccount where
-    build (W.ChimericAccount addr) = hexF addr
```

Comment:
Found this little orphan :upside_down_face: ... No longer needed since it's define in the core package next to its relevant type declaration.